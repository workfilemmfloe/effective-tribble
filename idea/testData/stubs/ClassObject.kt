class C { default object {
    fun foo() {
    }
}
}