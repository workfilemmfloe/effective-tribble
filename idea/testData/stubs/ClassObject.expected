PsiJetFileStubImpl[package=]
  PACKAGE_DIRECTIVE:
  CLASS:[fqName=C, isEnumEntry=false, isLocal=false, isTopLevel=true, isTrait=false, name=C, superNames=[]]
    CLASS_BODY:
      OBJECT_DECLARATION:[fqName=C.Default, isDefault=true, isLocal=false, isObjectLiteral=false, isTopLevel=false, name=Default, superNames=[]]
        MODIFIER_LIST:[default]
        CLASS_BODY:
          FUN:[fqName=C.Default.foo, hasBlockBody=true, hasBody=true, hasTypeParameterListBeforeFunctionName=false, isExtension=false, isProbablyNothingType=false, isTopLevel=false, name=foo]
            VALUE_PARAMETER_LIST:
